.util/pylint_scores.txt

```diff
@@ -1,4 +1,20 @@
 
+------------------------------------
+Your code has been rated at 10.00/10
+
+************* Module login
+Dashboard/login.py:15:0: C0301: Line too long (471/100) (line-too-long)
+
+-----------------------------------
+Your code has been rated at 9.84/10
+
+************* Module price
+Dashboard/pages/price.py:8:0: E0401: Unable to import 'navigation' (import-error)
```

Comment:
Please update `requirements.txt` to include this requirement.

Reply:
I see this is from `navigation.py`, so I'm not sure why pylint is complaining

Reply:
Yeah I’m not sure why. I know that the price.py file is in a child folder but I think it still works